### ISSUE TITLE
Doesn't show the thumbnail

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mcsavadhar/timthumb#62]
```
What steps will reproduce the problem?
1. I installed timthumb with correctly chmods in cache and temp folder


What is the expected output? What do you see instead?
The only thing that I see is the thumb's url

What version of the product are you using? On what operating system?
Version: http://www.binarymoon.co.uk/2009/07/timthumb-beta-test/

Please provide any additional information below.
I tried in two different hosts
http://folker.comoj.com/data/timthumb.php?src=http://i49.tinypic.com/b4b5vo.jpg&
h=60&w=100&zc=1
http://sogostosa.freetzi.com/tim/timthumb.php?src=http://4.bp.blogspot.com/_A9QL
FUJhKeY/SnJa6QgywQI/AAAAAAAAAGI/euPr9DQORrA/s1600/1248979324_carmen_pack_preview
.jpg&h=&w=&zc=1

```

Original issue reported on code.google.com by `folke...@hotmail.com` on 21 Jan 2010 at 7:05

**Comment by GoogleCodeExporter** [comment]
```
Same problem. I'm using PHP 5.3.0
```

Original comment by `Bogdan.O...@gmail.com` on 24 Jan 2010 at 10:51

**Comment by GoogleCodeExporter** [comment]
```
can anyone resolve this?
```

Original comment by `folke...@hotmail.com` on 1 Feb 2010 at 3:07

**Comment by GoogleCodeExporter** [comment]
```
Hi,
I have posted this to ISSUE 56, but you opened thread with same problem, so 
here is
the solution:
 I have just changed ereg to preg_match and used # delimiter. This blogpost helped me
with it a lot:
http://devthought.com/tumble/2009/06/fix-ereg-is-deprecated-errors-in-php-53/


FIX FOR PHP 5.3:
------------------------

find line 550, replace:

if (ereg('http://' $src) == true) {  

with:

if (preg_match('#http://#' $src) == true) {

// (using # delimiter)



then at line 556 replace:

if (ereg($site, $url_info['host']) == true) {

with:

if (preg_match($site, $url_info['host']) == true) {



*Working for me* @ three PHP 5.3 hostings and at my local EasyPHP 2.0 ;), I 
hope it
will work for you all as well.

```

Original comment by `steww...@gmail.com` on 14 Mar 2010 at 8:25

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
thanx to alesmana at issue 56 for noticing: comma isn't for some strange reason
displayed, so it is:

if (preg_match('#http://#'<comma> $src) == true) {

I'm attaching fixed version for lazy guys like me ;)


```

Original comment by `steww...@gmail.com` on 19 Apr 2010 at 4:56

Attachments:
- [timthumb-php5_3.php](https://storage.googleapis.com/google-code-attachments/timthumb/issue-62/comment-7/timthumb-php5_3.php)

**Comment by GoogleCodeExporter** [comment]
```
Hi, this fix for PHP 5.3 is now in the core release so just update to the latest
version on google code and you should be set.
```

Original comment by `BinaryMoon` on 26 Apr 2010 at 9:02
- Changed state: **Fixed**
